packages/paste-core/components/toggle/src/ToggleKnob.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import type {ToggleKnobProps} from './types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {CheckboxCheckIcon} from '@twilio-paste/icons/esm/CheckboxCheckIcon';
+
+const ToggleKnob = React.forwardRef<HTMLDivElement, ToggleKnobProps>(
+  ({element, disabled, toggleIsOn, isHovering, ...props}, ref) => {
+    const leftPlacement = (): string => {
+      if (toggleIsOn && isHovering && !disabled) return '22px';
+      else if (toggleIsOn && !isHovering) return 'space70';
+      else if (toggleIsOn && disabled) return 'space70';
+      else if (!toggleIsOn && isHovering && !disabled) return 'space20';
+      else return '2px';
+    };
```

Comment:
issue: this logic may fall apart within customization provider. What you may want instead is to use `translateX()` because it's more performant than animating/changing `left` in CSS and to set it to a value like `calc(100% - width of knob)`